eyieko/landville-frontend
# #168309097 Enable reply to reviews

## Description ##
- Enable users to reply to reviews

## Type of change ##
Please select the relevant option

## How Has This Been Tested? ##
Please describe the tests that you ran to verify your changes

## Checklist: ##

## How can this PR be manually tested: ##
- Navigate to `partners` page 
- Select read reviews
- Click on reply option under a specific review
- Upon successful submission of reply, one should get a message displaying the reply they have added to a review.

## Any background context you want to provide? ##
- As an authenticated user I want to be able to reply to am existing client review so I can provide my opinion.

## Related Pivotal Tracker stories ##
[#168309097](https://www.pivotaltracker.com/story/show/168309097)
